### ISSUE TITLE
Dagger 2.9 release is missing android-dagger classes

### ISSUE THREAD

**Issue by fSergio101** [google/dagger#581]
I'm trying to use android-dagger extension from release 2.9 and using the following reference from gradle:

```
compile 'com.google.dagger:dagger-android:2.9'
```

I got the lib downloaded but I think that there is something missing in the classes it contains, here you have the snapshot: 
![screen shot 2017-02-15 at 17 22 52](https://cloud.githubusercontent.com/assets/446755/22983757/6c74d826-f3a3-11e6-9cec-caccf6171f1d.png)

Comparing with the repo branch [google/dagger/blob/prepare_release_2_9](https://github.com/google/dagger/tree/prepare_release_2_9) it contains the same files, almost nothing. On the other hand, [master](https://github.com/google/dagger) looks to be fine, because it contains all the files that are even described in [dagger & Android](https://google.github.io/dagger/android.html) documentation. That makes me think that maybe about some bad merge or something like that.

Extra point: I know that maybe this is not the best place to notify this, but the [dagger & Android](https://google.github.io/dagger/android.html) documentation I mentioned above has some broken links (the ones referencing to https://google.github.io/dagger/api/latest/dagger/android/* ) and markdown typos.

**Comment by ronshapiro** [comment]
The classes in the documentation will be released in the next version of dagger. Sorry for the confusion that the docs are a little bit ahead

**Comment by naturalwarren** [comment]
I was able to use the dagger-android artifact just fine in 2.9 but did run into https://github.com/google/dagger/issues/579

**Comment by trajakovic** [comment]
But again, whole artifact (dagger-android) is missing on sonatype snapshot repo (HEAD-SNAPSHOT). Is this "on purpose"?

**Comment by ronshapiro** [comment]
No, and it will be fixed shortly. Why do you need the snapshot though?

**Comment by trajakovic** [comment]
@ronshapiro I was expecting that missing (dagger-android) classes in 2.9 mentioned by @fSergio101 (and documentation) will be in "HEAD-SNAPSHOT" build...

**Comment by ronshapiro** [comment]
This should all be fixed in `2.10-rc2`
